go/kbnm/handler.go

```diff
@@ -57,24 +57,24 @@ func checkUsernameQuery(s string) (string, error) {
 }
 
 // Handler returns a request handler.
-func Handler() *handler {
-	return &handler{
+func GetHandler() *Handler {
```

Comment:
FWIW I disagree with the linter warning in this case from a semantic perspective, but I assume we're not using a configurable linter so I guess the change it necessary. :/

IIRC it was something along the lines of "returning unexported concrete types which can be annoying to work with" or something silly like that, particularly since this is a `package main` and the package has full scope to access it and it can't be imported from anywhere else so it's pretty moot.

I believe just making `func Handler() ...` unexported will calm the linter down too. (e.g. `func newHandler() *handler`).

Anyways, I don't care supermuch about this, feel free to just keep this change if it's easier.